src/API/Endpoints/Reports/Endpoint.php

```diff
@@ -117,6 +128,8 @@ private function setupProperties( $request ) {
 		$this->request   = $request;
 		$this->startDate = date_create( $request->get_param( 'start' ) );
 		$this->endDate   = date_create( $request->get_param( 'end' ) );
+		$this->currency  = $request->get_param( 'currency' );
```

Comment:
@henryholtgeerts you did not create a `currency` and `testMode` as the property of the class.

Reply:
Done! ref: https://github.com/impress-org/givewp/commit/df39b2e7da52361bfc9368f8b4fc30fc5fc7f826